src/Raven.Server/Documents/Subscriptions/SubscriptionProcessor/DocumentsSubscriptionProcessor.cs

```diff
@@ -223,6 +223,11 @@ private bool ShouldAddToResendTable(DocumentsStorage.DocumentOrTombstone item, s
                 switch (status)
                 {
                     case ConflictStatus.Update:
+                        // check if the docs still matches criteria
+                        var result = item.Document;
```

Comment:
should this https://github.com/ravendb/ravendb/blob/de703f22a1bc67ce5f52d25a0552dd05dee8d00f/src/Raven.Server/Documents/Subscriptions/SubscriptionProcessor/DocumentsSubscriptionProcessor.cs#L172 
handle this case?

Reply:
Yes, I would expect that this would be filtered on the fetcher as well.

Reply:
done